python/cudf_polars/tests/test_join.py

```diff
@@ -69,9 +69,9 @@ def test_join_maintain_order_param_unsupported(left, right, maintain_order):
         ["c", "a"],
     ],
 )
-def test_non_coalesce_join(left, right, how, join_nulls, join_expr):
+def test_non_coalesce_join(left, right, how, nulls_equal, join_expr):
     query = left.join(
-        right, on=join_expr, how=how, join_nulls=join_nulls, coalesce=False
+        right, on=join_expr, how=how, nulls_equal=nulls_equal, coalesce=False
```

Comment:
This change breaks the tests for Polars<1.25 (sorry if this is already known, just saw the [nightly failures](https://github.com/rapidsai/cudf/actions/runs/13805122307/job/38614356633) and started looking for the cause).

Reply:
Thanks, no I wasn't aware. I wonder why polars==1.23 is installed? It should be 1.25